test/perf/merge.js

```diff
@@ -44,35 +44,35 @@ var options = {
 
 suite
   .add('most', function(deferred) {
-    var streams = a.map(most.from);
-    runners.runMost(deferred, most.mergeArray(streams).reduce(sum, 0));
+    const streams = map(from, a);
```

Comment:
`a` is an Array here, so this line should be: `const streams = a.map(from)`.  That's why the test is bailing out immediately--not sure why the failure is silent, tho :/